### ISSUE TITLE
Support for scheduled jobs in the meltano API & UI

### ISSUE THREAD

**Issue by pandemicsyn** [meltano/meltano#6176]
Spinning out [this thread](https://gitlab.com/meltano/meltano/-/issues/3479#note_961801123) from https://github.com/meltano/meltano/issues/3399 as a dedicated issue.

The meltano API has a `/v1/orchestrations/pipeline-schedules` path that supports Adding, Updating, Deleting, and Listing meltano schedules. To roll out `jobs` in `schedules` support in time for 2.0 we didn't fully add support for jobs in the API. We adjusted the API just enough to continue to allow adding and updating existing scheduled legacy ELT tasks via the UI & API. That means you can't see nor add or modify scheduled job's in the UI. Deletion of scheduled jobs via the API should already work, since that's just based on the schedule name.

Full listing with jobs included is actually possible if the feature flag `ENABLE_API_SCHEDULED_JOB_LIST` is toggled, but if we want to add full support for jobs in the schedules UI, we'll need to update the `/v1/orchestrations/pipeline-schedules` endpoint to support adding, updating schedules with jobs as well as update the UI itself to support the jobs concept. 

If we would like to add a new UI to support viewing/editing jobs themselves's we'd of course also need a new API endpoint and CRUD ops to manage them.

**Comment by pandemicsyn** [comment]
Realizing a few weeks after logging this that we should not actually shoe-horn this into the v1 API and wait on the v2 API. 

https://github.com/meltano/meltano/blob/main/docs/adr/0005-use-fastapi-to-develop-v2-of-api.md#consequences

> No new features will be introduced in the v1 API, and it will effectively enter a maintenance only mode (the UI will likely continue to receive quality of life improvements backed by the v1 API for a period of time).

(Nearly suggested violating an ADR I logged myself 🤡)

**Comment by WillDaSilva** [comment]
Closing as no longer relevant on account of the removal of the UI for Meltano 3.0
